webpack.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/server.js',
```

Comment:
because we thought it is connected to everything hbs and most backend plus we saw it on a video.